magento/magento2
# Fixed 19814: Admin Forgot password captcha Icon misaligned.

### Description (*)

Admin Forgot password captcha Icon misaligned.
### Fixed Issues (if relevant)

1. magento/magento2#19814: Admin Forgot password captcha Icon misaligned. 

### Manual testing scenarios (*)

1. Hit magento admin url > Click on Forgot password > Check captcha refresh icon alignment.

### Contribution checklist (*)
